HW_5/app_HW_5.py

```diff
@@ -0,0 +1,64 @@
+'''Необходимо создать API для управления списком задач. Каждая задача должна содержать заголовок и описание. Для каждой задачи должна быть возможность указать статус (выполнена/не выполнена).
+
+API должен содержать следующие конечные точки:
+— GET /tasks — возвращает список всех задач.
+— GET /tasks/{id} — возвращает задачу с указанным идентификатором.
+— POST /tasks — добавляет новую задачу.
+— PUT /tasks/{id} — обновляет задачу с указанным идентификатором.
+— DELETE /tasks/{id} — удаляет задачу с указанным идентификатором.
+
+Для каждой конечной точки необходимо проводить валидацию данных запроса и ответа. Для этого использовать библиотеку Pydantic.'''
+
+from fastapi import FastAPI
+from fastapi.responses import HTMLResponse
+import pandas as pd
+from flask import Request
+from pydantic import BaseModel
+from fastapi.templating import Jinja2Templates
+
+app = FastAPI()
+templates = Jinja2Templates(directory="./HW_5/templates")
+
+tasks = []
+
+
+class Task(BaseModel):
+    id: int
+    title: str
+    description: str
+    status: bool = False
+
+
+@app.get('/task', response_class=HTMLResponse)
+async def show_tasks(request: Request):
+    table = pd.DataFrame([vars(task) for task in tasks]).to_html()
+    return templates.TemplateResponse('task_HW_5.html', {"request":request,"table":table})
+
+@app.get('/task/{task_id}', response_class=HTMLResponse)
+async def show_task(request: Request,task_id:int):
+    task = pd.DataFrame([vars(my_task) for my_task in tasks if my_task.id ==- task_id]).to_html(index=False)
```

Comment:
Что-то лишний символ затесался, по моему
```suggestion
    task = pd.DataFrame([vars(my_task) for my_task in tasks if my_task.id == task_id]).to_html(index=False)
```